lib/matplotlib/tests/test_backends_interactive.py

```diff
@@ -50,16 +50,14 @@ def _get_testable_interactive_backends():
             reason = "macosx backend fails on Azure"
         elif 'qt5' in backend and not have_qt5:
             reason = "no usable Qt5 bindings"
+        marks = []
         if reason:
-            backend = pytest.param(
-                backend,
-                marks=pytest.mark.skip(
-                    reason=f"Skipping {backend} because {reason}"))
+            marks.append(pytest.mark.skip(
+                reason=f"Skipping {backend} because {reason}"))
         elif backend.startswith('wx') and sys.platform == 'darwin':
             # ignore on OSX because that's currently broken (github #16849)
-            backend = pytest.param(
-                backend,
-                marks=pytest.mark.xfail(reason='github #16849'))
+            marks.append(pytest.mark.xfail(reason='github #16849'))
+        backend = pytest.param(backend, marks=marks)
```

Comment:
This change ensures we always have a `pytest.param`, so that the later loop is simpler.

Reply:
Yeah, I can't not read.